### PR TITLE
Fev 293 open question feedback

### PR DESCRIPTION
**Pull request by semarche-kaltura**
Screen Shot 2019-06-15 at 10 00 15 PM
Screen Shot 2019-06-15 at 10 00 39 PM

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @semarche-kaltura,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: modules/Quiz/resources/quiz.js [hunk 1]
```diff
@@ -790,9 +790,25 @@
             if($.cpObject.cpArray[selectedQuestion].questionType === this.KIVQModule.QUESTIONS_TYPE.OPEN_QUESTION 
                 && $.cpObject.cpArray[selectedQuestion].openAnswer){
                 var text = $.cpObject.cpArray[selectedQuestion].openAnswer;
+                var feedback = $.cpObject.cpArray[selectedQuestion].feedback;
                 text = text.replace(/\n/g,"<br/>");
                 $(".yourAnswer").html(text).addClass("open-question-answer");
                 $(".correctAnswerText").html("");
+                if (feedback) {
+                    console.log($.cpObject)
```
Comment (eitanavgil) [inline modules/Quiz/resources/quiz.js]: :)

File: modules/Quiz/resources/quiz.js [hunk 2]
```diff
@@ -790,9 +790,25 @@
...
+                    $(".feedback").html("Feedback").bind('click', function () {
+                        $(".feedback-modal").show();
+                        $(".reviewAnswerPlace").hide();
+                        $(".reviewAnswerFooter").hide();
```
Comment (eitanavgil) [inline modules/Quiz/resources/quiz.js]: $(".feedback").html("Feedback").off('click') or use unbind

Comment (eitanavgil) [reply modules/Quiz/resources/quiz.js]: $(".feedback").html("Feedback").off().bind(....) // or maybe it requires on("mouseClick")

File: modules/Quiz/resources/quiz.js [hunk 3]
```diff
@@ -790,9 +790,25 @@
...
+
+                    });
+                    $(".feedback-modal .icon-close").bind('click', function () {
```
Comment (eitanavgil) [inline modules/Quiz/resources/quiz.js]: Accessibility features: 

1 make sure that after pop-up we need to set the focus to the X button (JQ focus) 
tab loops (might need manual management) between close and text 
need to be able to open the feedback with KB 
Need to be able to close the pop-up using tab and KB only (space OR enter)
Make sure that the professor feedback is accessible to screen-readers.

File: modules/Quiz/resources/IVQScreenTemplate.js [hunk 4]
```diff
@@ -63,11 +63,16 @@
                 $(".icon-close").css("display", "none");
                 $(_this.emptyScreen()).hide().append(
                     '<div class="ivqContainer">'+
+                    '   <div class="feedback-modal">' +
+                    '       <span class="icon-close" aria-label="Close feedback" tabindex="0" role="button"></span>' +
```
Comment (eitanavgil) [inline modules/Quiz/resources/IVQScreenTemplate.js]: Locale - try to see if we can pass this via the translation function 
Also applies for the "feedback" button

File: modules/Quiz/resources/quiz.js [hunk 5]
```diff
@@ -790,9 +790,25 @@
             if($.cpObject.cpArray[selectedQuestion].questionType === this.KIVQModule.QUESTIONS_TYPE.OPEN_QUESTION 
                 && $.cpObject.cpArray[selectedQuestion].openAnswer){
                 var text = $.cpObject.cpArray[selectedQuestion].openAnswer;
+                var feedback = $.cpObject.cpArray[selectedQuestion].feedback;
                 text = text.replace(/\n/g,"<br/>");
                 $(".yourAnswer").html(text).addClass("open-question-answer");
                 $(".correctAnswerText").html("");
+                if (feedback) {
+                    console.log($.cpObject)
+                    $(".feedback").html("Feedback").bind('click', function () {
```
Comment (eitanavgil) [inline modules/Quiz/resources/quiz.js]: Locale this

**Pull request merged**